docs/software/desktop.md

```diff
@@ -172,24 +172,92 @@ Website: <http://gnustep.org>
 
 ### Chromium
 
-#### Does my device support GPU acceleration?
+![Chromium screenshot](../assets/images/dietpi-software-desktop-tools-chromium.jpg){: width="1376" height="858" loading="lazy"}
 
-Not all devices support GPU acceleration with Chromium.
+=== "Chromium kiosk mode"
 
-- RPis supports native OpenGL and will be configured during installation.
-- Odroids support OpenGLES and will be configured during installation.
-- Other devices will most likely be limited to software rendering.
+    <h4>Overview</h4>
 
-#### Where can I check the status of GPU acceleration?
+    The **Chromium kiosk mode** is a Chromium feature that allows your device to be used in a single-window resp. single-app kiosk mode. Generally, any access to the rest of your system is locked.  
+    The mode can be used to set the web browser to these properties:
 
-Once Chromium is running, simply go to the following address: `chrome://gpu`
+    - Full screen
+    - No menu bar, no toolbar, no address field,...
+    - No option to exit the browser resp. change the tabs
 
-#### Enable support for Widevine DRM protected content on RPi
+    Two examples using the mode are to use your system
 
-- Manually: [https://blog.vpetkov.net/2019/07/12/net ... -chromium/](https://blog.vpetkov.net/2019/07/12/netflix-and-spotify-on-a-raspberry-pi-4-with-latest-default-chromium/)
-- with scripts: [https://blog.vpetkov.net/2020/03/30/ras ... many-others/](https://blog.vpetkov.net/2020/03/30/raspberry-pi-netflix-one-line-easy-install-along-with-hulu-amazon-prime-disney-plus-hbo-spotify-pandora-and-many-others/)
+    - as a display of only one or several web pages (via tabs) without access to the whole system
+    - as a browser based smart home GUI
 
-![Chromium screenshot](../assets/images/dietpi-software-desktop-tools-chromium.jpg){: width="1376" height="858" loading="lazy"}
+    The kiosk mode can be distinguished to the **Single App Kiosk Mode**, where the user is locked into one single particular site having a fullscreen mode that can't be changed or exited.  
+    Following is an example of a kiosk mode screenshot for the example site `dietpi.com`:
+
+    ![Chromium kiosk mode screenshot](../assets/images/dietpi-software-desktop-chromium-kiosk-mode.jpg){: width="360" height="208" loading="lazy"}
+
+    <h4>Activate Chromium kiosk mode</h4>
+
+    To put your Chromium browser into kiosk mode the following steps have to be achieved:
+
+    1. Install Chromium browser via 
+
+        ```sh
+        dietpi-software install 113
+        ```
+
+        If Chromium is not installed, you will get a startup error message that the file `/var/lib/dietpi/dietpi-software/installed/chromium-autostart.sh` is not found if you activate Chromium kiosk mode via `dietpi-autostart`.
+
+    1. Select Chromium kiosk mode (with automatic login) via
+
+        ```sh
+        dietpi-autostart
```

Comment:
Or shall we document the quick option:
```sh
dietpi-autostart 11
```
and
```sh
dietpi-autostart 0
```
to revert back to manual console login?

Reply:
This will not work because then you are not asked about the web page to display and the login user.

Reply:
Ah right, I always forget about the interactive parts. Actually we should merge those so that all interactive dialogues also show up with the quick option. In non-interactive calls/environment, those dialogues are cancelled anyway, so it shouldn't introduce any regressions (but need to double check all calls).